patchnetvlad/tools/msls.py

```diff
@@ -0,0 +1,549 @@
+import numpy as np
```

Comment:
This file requires a license header and reference to https://github.com/mapillary/mapillary_sls on which the file is heavily based on.

Reply:
(best to copy their license header, and add a "modified by ..")

Reply:
Done